tests/drivers/uart/uart_elementary/testcase.yaml

```diff
@@ -0,0 +1,58 @@
+common:
+  tags: drivers uart
+  depends_on: gpio
+  harness: ztest
+  harness_config:
+    fixture: gpio_loopback
+tests:
+  drivers.uart.uart_elementary:
+    filter: CONFIG_SERIAL_SUPPORT_INTERRUPT
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf54l15pdk/nrf54l15/cpuapp
+      - nrf5340dk/nrf5340/cpuapp
+    extra_configs:
+      - CONFIG_UART_INTERRUPT_DRIVEN=y
+      - CONFIG_UART_0_INTERRUPT_DRIVEN=y
```

Comment:
nrf54s don't have instance 0. Shouldn't those 2 entries be removed?

Reply:
How to enable interrupt driven UART api for nrf54h with K-configs ?